scopes/design/ui/contributors/contributors.tsx

```diff
@@ -1,6 +1,6 @@
 import { UserAvatar } from '@teambit/ui.avatar';
 import { TimeAgo } from '@teambit/ui.time-ago';
-import { AccountObj } from '@teambit/ui.version-block';
+import { AccountObj } from '@teambit/ui.avatar';
```

Comment:
Did you make this change?

Reply:
Yes - it's related to the change above re double declaration of the same account type+enum. I made the declarations in 'avatar' the only ones and redirected this file to have avatar as a dep to get the declarations from there

Reply:
@KutnerUri @odedre - can one of you please review this

Reply:
ah I see. Contributors should have probably used <Avatar /> to begin with.
Reusing the type makes sense